services/resource-usage-tracker/src/simcore_service_resource_usage_tracker/resource_tracker_process_messages.py

```diff
@@ -63,12 +63,25 @@ async def _process_start_event(
     msg: RabbitResourceTrackingStartedMessage,
     rabbitmq_client: RabbitMQClient,
 ):
+    service_run_db = await resource_tracker_repo.get_service_run_by_id(
+        service_run_id=msg.service_run_id
+    )
+    if service_run_db:
+        # NOTE: After we find out why sometimes RUT recieves multiple start events and fix it, we can change it to log level `error`
+        _logger.warning(
+            "On process start event the service run id %s already exists in DB, INVESTIGATE! Current msg created_at: %s, already stored msg created_at: %s",
```

Comment:
instead of messages like INVESTIGATE, shouldnt' we define some constant keys that we can use to trigger alarms. E.g. `ALARM_MARK_RED`, `ALARM_MARK_YELLOW` , etc then we can use graylog to track these.

Reply:
don't you have that OEC thingy @pcrespov?

Reply:
OECs are for the front end, these errors will not be propagated to the user. Regarding alarms, I think any error should trigger an "alarm" that's for example also the reason why I have caught the known error and translated it to a warning, as it is ok for now.